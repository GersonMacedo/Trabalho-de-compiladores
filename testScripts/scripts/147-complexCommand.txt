! -s SYNTACTIC
! should accept a mix of all types of commands
program p ;
begin
    begin
        begin
            a := false;
            if true then
                while false do
                    b := b + 1
            else
                while false do
                    b := b - 1
            ;
            while t do
                a := a and a
            ;
        end ;
        a := true;
    end ;
    c := 5;
    if false then
        while t do
        begin
            a := true;
            b := 4;
        end
    ;
    while k do
        if t then
            a := false
        else
            a := true
    ;
end .