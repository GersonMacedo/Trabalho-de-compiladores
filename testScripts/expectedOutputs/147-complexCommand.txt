
Performing up to syntactic analysis at testScripts/scripts/147-complexCommand.txt
No errors found
